pkg/rtc/transportmanager.go

```diff
@@ -609,3 +623,32 @@ func (t *TransportManager) ProcessPendingPublisherDataChannels() {
 		}
 	}
 }
+
+func (t *TransportManager) OnReceiverReport(dt *sfu.DownTrack, report *rtcp.ReceiverReport) {
+	t.mediaLossProxy.HandleMaxLossFeedback(dt, report)
+}
+
+func (t *TransportManager) onMediaLossUpdate(loss uint8) {
+	t.udpLossUnstableCount <<= 1
+	if loss >= uint8(255*udpLossFracUnstable/100) {
+		t.udpLossUnstableCount |= 1
+		if bits.OnesCount32(t.udpLossUnstableCount) >= udpLossUnstableCountThreshold {
+			if t.udpRTT > 0 && t.tcpRTT < uint32(float32(t.udpRTT)*1.3) && t.tcpRTT < tcpGoodRTT {
```

Comment:
Should we actually check against `min` of UDP RTT? Or some smoothed version of it? This is checking against the latest RTT when the count crosses the threshold. Wondering if that is going to be erratic.

Reply:
Added a smooth to RTT update.  rtt += (newRtt - oldRtt)/2